### PR TITLE
Change publish interval

### LINKED ISSUE #14: Change default update interval
As written in some earlier issue, for me having the 10s update interval locked the ModBus interface on the unit itself from time to time. After changing the configuration in HA to 60s it has worked flawlessly. 

Should we change the default to 60 instead of 10? Affected file config.json

### PR DESCRIPTION
**Pull request by tomrosenback**
Fixes #14

### CODE REVIEW

File: eda-modbus-bridge/DOCS.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ mqtt:
   port: 1883
   username: ''
   password: ''
-  publish_interval: 10
+  # Publish interval in seconds. Any value should work here but on some devices to low interval might lock up the
```
Comment (Jalle19) [inline eda-modbus-bridge/DOCS.md]: too low*

File: eda-modbus-bridge/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
   * Configure entity icons for alarm sensors
   * Move Home Assistant related logic to `homeassistant.mjs`
   * Publish device information over MQTT once only (https://github.com/Jalle19/eda-modbus-bridge/issues/43)
+- Change default publish interval to 60s, see [DOCS.md](https://github.com/Jalle19/home-assistant-addon-repository/blob/main/eda-modbus-bridge/DOCS.md) for more info (https://github.com/Jalle19/home-assistant-addon-repository/issues/14)
```
Comment (Jalle19) [inline eda-modbus-bridge/CHANGELOG.md]: I pulled the trigger on the release already, can you move this to a new 1.2.1 release?

Comment (tomrosenback) [reply eda-modbus-bridge/CHANGELOG.md]: I noted that myself too, but, as this addon is not released as such, any change in the main branch comes out HA aswell, right? This mean my change in itself here would be a "release". Should I change all the versioning aswell? In other words we will get 1.2.1 available once merged?

Comment (tomrosenback) [reply eda-modbus-bridge/CHANGELOG.md]: Increased version already.

Comment (Jalle19) [reply eda-modbus-bridge/CHANGELOG.md]: Yeah I think it's best to make a new release either way, the Dockerfile pulls a specific tag so it's not enough to just push something to master.

Comment (tomrosenback) [reply eda-modbus-bridge/CHANGELOG.md]: From the eda-modbus-bridge repo yes. But this repo has no tags, HA pulls latest only.

Comment (Jalle19) [reply eda-modbus-bridge/CHANGELOG.md]: Yeah

**Review by Jalle19 — APPROVED** [review]

**Comment by Jalle19** [comment]
Thanks!

**Comment by tomrosenback** [comment]
no problem

**Pull request merged**